mesa-libgbm-devel-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: mesa-libgbm-cos6-x86_64
```

Comment:
```suggestion
  name: mesa-libgbm-devel-cos6-x86_64
```
It should be this instead, right?

Reply:
That's what I get for thinking this file looked too simple to warrant using find/replace.

Reply:
There is actually a script to generate these files. It's either in `conda skeleton` or in grayskull. Can't remember which.

Reply:
The recipes looked pretty boilerplate from one to the next. Anything tricky to watch out for that the tool would catch for me?

Reply:
Typos and naming convention AFAIK.

Reply:
you're getting `ERROR: Glob /feedstock/recipe did not match in root_dir /feedstock/.abs`. Could be related?